docs/080-breaking-changes.rst

```diff
@@ -61,3 +61,8 @@ New Restrictions
 
 * Declarations with the name ``this``, ``super`` and ``_`` are disallowed, with the exception of
   public functions and events.
+
+Command Line and JSON Interfaces
+================================
+
+* Changed output of `--combined-json`. Content of JSON-Fields `abi`, `devdoc`, `userdoc` and `storage-layout` are sub-objects now. Before 0.8.0 they were appeared as serialised to strings.
```

Comment:
In reStructuredText you need double backticks to get the same effect as in Markdown. I also tweaked the wording a bit:

```suggestion
* Changed output of ``--combined-json``. JSON fields ``abi``, ``devdoc``, ``userdoc`` and ``storage-layout`` are sub-objects now. Before 0.8.0 they used to be serialised as strings.
```